app/components/time-picker.js

```diff
@@ -2,11 +2,24 @@ import moment from 'moment';
 import Ember from 'ember';
 import momentFormat from 'ember-moment/computeds/format';
 
+const { computed } = Ember;
+const { oneWay } = computed;
+
 export default Ember.Component.extend({
   classNames: ['time-picker'],
   date: null,
-  dateBuffer: Ember.computed.oneWay('date'),
-  hour: momentFormat('dateBuffer', 'h'),
+  dateBuffer: oneWay('date'),
+
+  hour: computed('dateBuffer', function() {
+    const defaultHour = this.get('defaultHour');
```

Comment:
This was extremely tricky to pull off. It doesn't let me do both: set a visible formatted date on the start-date input AND have the time data passed down to reflect the default time. When the date gets formatted to "MM/DD/YYYY", it resets the hours and minutes.